### PR TITLE
removing job-scheduler zip for 1.3 branch

### PR DESCRIPTION
**Pull request by amitgalitz**
Signed-off-by: Amit Galitzky <amgalitz@amazon.com>

### Description
Draft PR to try out removing job-scheduler zip for integ testing and bwc testing for current version.
This is cut against 1.3 branch since 2.0 branch currently is having issue with version bump plus unreleased docker image and etc.
 
### Issues Resolved
https://github.com/opensearch-project/anomaly-detection/issues/17
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,12 +15,13 @@ import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
 
 buildscript {
     ext {
+        job_scheduler_version = "1.3.1"
```
Comment (ohltyler) [inline build.gradle:18]: Can we generate this value from the `opensearch_version` on line 20? The fewer places where we have to manually change during version bump, the better.

File: build.gradle [hunk 2]
```diff
@@ -335,7 +342,13 @@ List<Provider<RegularFile>> plugins = [
                 return new RegularFile() {
                     @Override
                     File getAsFile() {
-                        return fileTree(bwcFilePath + "job-scheduler/" + project.version).getSingleFile()
+                        project.mkdir "src/test/resources/job-scheduler"
```
[the remark this answers is not in the document]
Comment (amitgalitz) [reply build.gradle]: Yeah you are right its not needed for this

**Review by ohltyler — APPROVED** [review]

**Comment by amitgalitz** [comment]
@ohltyler in terms of 1.x, it does seem like infra hasn't ran that build since march 23 for core or anything else so job-scheduler and common utils don't have a 1.4 built yet.

**Comment by ohltyler** [comment]
> @ohltyler in terms of 1.x, it does seem like infra hasn't ran that build since march 23 for core or anything else so job-scheduler and common utils don't have a 1.4 built yet.

Yeah, not sure, was supposed to automatically build after I added them to the manifest but I was told there's some bug causing it to not run - could be because there is no 1.4 test manifest. I've kicked off a 1.4 build manually and we'll see what happens.

**Review by Zhangxunmt — APPROVED** [review]

**Pull request merged**